vm.go

```diff
@@ -341,6 +426,9 @@ func (vm *vm) newStash() {
 }
 
 func (vm *vm) init() {
+	vm.sb = -1
+	vm.stash = &vm.r.global.stash
+	vm.maxCallStackSize = math.MaxInt32
```

Comment:
At line 472 we try to access:

```go
vm.prg.code[vm.pc].exec(vm)
```

but `vm.pc` can be `-1` which would cause a panic.